.rubocop_todo.yml

```diff
@@ -87,7 +87,7 @@ RSpec/InstanceVariable:
 # Offense count: 38
 # Configuration parameters: AggregateFailuresByDefault.
 RSpec/MultipleExpectations:
-  Max: 29
+  Max: 36
```

Comment:
manually tweaked since the 3d test has more expectations now

Reply:
In other repositories we disabled ` RSpec/MultipleExpectations` in rubocop.yml

Reply:
adjusting rubocop config